### PR TITLE
DP-28303: Entity Usage regenerate new command

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-28303

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against de0259090f63f96d0b6e5992367995eb3c19daff
</p>

File: docroot/modules/custom/mass_entity_usage/src/Commands/MassEntityUsageCommands.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+   *
+   * @command mass-content:usage-regenerate
+   * @aliases maur,mass-usage-regenerate
+   * @option batch-size
+   *   When --use-queue is used, the queue will be populated in a batch process
+   *   to avoid memory issues. The --batch-size flag can be optionally used to
+   *   specify the batch size, for example --batch-size=500.
+   */
+  public function recreate($options = ['batch-size' => 1000]) {
+    $this->database->delete('queue')->condition('name', 'entity_usage_tracker')->execute();
```
Comment (arthurbaghdas) [inline docroot/modules/custom/mass_entity_usage/src/Commands/MassEntityUsageCommands.php]: Reminder to rename this

Comment (arthurbaghdas) [reply docroot/modules/custom/mass_entity_usage/src/Commands/MassEntityUsageCommands.php]: We are using queue_unique table for this one. Fixed

**Comment by tom-fleming** [comment]
LGTM. Merging since we already ran on feature1

**Pull request merged**